dom0/securedrop-updater-cli

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+"""
+Utility script for checking and applying SecureDrop Workstation VM updates.
+
+The update process is intended to be called by the SecureDrop launcher, which
+is opened by the user when clicking on the desktop.
+"""
+
+import argparse
+import logging
+import subprocess
+import sys
+
+sdlog = logging.getLogger(__name__)
+# these are the TemplateVMs that require full patch level at boot in order to start the client
+possible_vms = ["dom0", "sd-svs-disp", "fedora", "airgapped"]
+# these are the TemplateVMs that for which the AppVMs are airgapped.
+# The Qubes Update manager cannot check for updates, we must do so manually.
+airgapped_vms = ["sd-svs-buster-template", "sd-log-buster-template", "securedrop-workstation-buster"]
+current_fedora_template = "fedora-30"
+current_sd_svs_disp_template = "sd-svs-disp-buster-template"
+
+UPDATES_OK = 0
+UPDATES_REQUIRED = 1
+UPDATES_FAILED = 2
+REBOOT_REQUIRED = 3
+
+
+def main(argv):
+    args = parse_argv(argv)
+    setup_logger(args.verbose)
+    sdlog.debug(args)
+    if args.check:
+        if "dom0" in args.check:
+            sdlog.debug("Checking for dom0 updates")
+            sys.exit(check_updates_dom0())
+        elif "sd-svs-disp" in args.check:
+            sdlog.debug("Checking for sd-svs-disp updates")
+            sys.exit(check_updates_sd_svs_disp())
+        elif "fedora" in args.check:
+            sdlog.debug("Checking for fedora updates")
+            sys.exit(check_updates_fedora())
+        elif "airgapped" in args.check:
+            sdlog.debug("Checking for updates for airgapped VMs")
+            rc = check_updates_airgapped()
+            try:
+                subprocess.Popen("qubes-update-gui")
```

Comment:
am i right that `qubes-update-gui` will open even if updates aren't needed currently?